src/MainWindow.cpp

```diff
@@ -166,30 +181,38 @@ void MainWindow::onFullscreen()
     m_fullscreen ? unfullscreen() : fullscreen();
 }
 
-bool MainWindow::onCloseToTray(bool visible, Gtk::Switch* startInTraySwitch)
+void MainWindow::onCloseToTray(Gtk::ModelButton* closeToTrayButton, Gtk::ModelButton* startInTrayButton)
 {
+    gboolean visible = closeToTrayButton->property_active();
```

Comment:
Please use `auto` if possible. Otherwise you can cast rhs to `bool`.
Something like this maybe makes more sense;
```cpp
auto const visible = !closeToTrayButton->property_active();
```